From 46a714008e89f86ba84eca65c80ee727d51414b2 Mon Sep 17 00:00:00 2001
From: Moritz Warning <moritzwarning@web.de>
Date: Thu, 12 Apr 2018 17:11:49 +0200
Subject: [PATCH 3/5] gcc48

---
 ext/json/json.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/json/json.hpp b/ext/json/json.hpp
index 9d48e7a6..bc5345aa 100644
--- a/ext/json/json.hpp
+++ b/ext/json/json.hpp
@@ -64,7 +64,7 @@ SOFTWARE.
     #endif
 #elif defined(__GNUC__)
     #define GCC_VERSION (__GNUC__ * 10000 + __GNUC_MINOR__ * 100 + __GNUC_PATCHLEVEL__)
-    #if GCC_VERSION < 40900
+    #if GCC_VERSION < 40800
         #error "unsupported GCC version - see https://github.com/nlohmann/json#supported-compilers"
     #endif
 #endif
-- 
2.16.3

